src/Lykke.Service.Assets/Cache/AssetsForClientCacheManager.cs

```diff
@@ -0,0 +1,172 @@
+            }
+        }
+
+        public async Task SaveSwiftDepositEnabledForClient(string clientId, bool isIosDevice, bool swiftDepositEnabled)
+        {
+            try
+            {
+                await _cache.SetStringAsync(GetKeySwiftDepositEnabled(clientId, isIosDevice), swiftDepositEnabled.ToJson(),
+                    new DistributedCacheEntryOptions() { AbsoluteExpirationRelativeToNow = _settings.AssetsForClientCacheTimeSpan });
+            }
+            catch (Exception ex)
+            {
+                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(SaveCashInViaBankCardEnabledForClient), clientId, ex);
+            }
+        }
+
+        public async Task<IReadOnlyList<string>> TryGetAssetForClient(string clientId, bool isIosDevice)
+        {
+            try
+            {
+                var json = await _cache.GetStringAsync(GetKeyAvailableAssets(clientId, isIosDevice));
+                if (!string.IsNullOrEmpty(json))
+                {
+                    return json.DeserializeJson<List<string>>();
+                }
+            }
+            catch (Exception ex)
+            {
+                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(SaveCashInViaBankCardEnabledForClient), clientId, ex);
+            }
+            return null;
+        }
+
+        public async Task<bool?> TryGetSaveCashInViaBankCardEnabledForClient(string clientId, bool isIosDevice)
+        {
+            try
+            {
+                var json = await _cache.GetStringAsync(GetKeyCashInViaBankCardEnabled(clientId, isIosDevice));
+                if (!string.IsNullOrEmpty(json))
+                {
+                    return json.DeserializeJson<bool>();
+                }
+            }
+            catch (Exception ex)
+            {
+                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(SaveCashInViaBankCardEnabledForClient), clientId, ex);
+            }
+            return null;
+        }
+
+        public async Task<bool?> TryGetSaveSwiftDepositEnabledForClient(string clientId, bool isIosDevice)
+        {
+            try
+            {
+                var json = await _cache.GetStringAsync(GetKeySwiftDepositEnabled(clientId, isIosDevice));
+                if (!string.IsNullOrEmpty(json))
+                {
+                    return json.DeserializeJson<bool>();
+                }
+            }
+            catch (Exception ex)
+            {
+                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(SaveCashInViaBankCardEnabledForClient), clientId, ex);
+            }
+            return null;
+        }
+
+        private string GetKeyAvailableAssets(string clientId, bool isIosDevice)
+        {
+            return string.Format(":Assets:Client:Available_Assets:{0}_{1}", clientId, isIosDevice ? "ios_device" : "");
```

Comment:
Could you use interpolated string here and below?

Reply:
it's same, but this paramert not simple 
isIosDevice ? "ios_device" : ""

Reply:
Seems good to me
return $":Assets:Client:Available_Assets:{clientId}_{(isIosDevice ? "ios_device" : string.Empty)}";